docs/plugins.rst

```diff
@@ -64,7 +72,14 @@ Once your config class is defined, you associate it with the plugin via the ``co
 Storage / Persisting Data
 --------------------------
 
-BasePlugin provides a wrapper around the Redis API that Plugins should use for storage. Since multiple channels can be using the same plugin, keys need to unique per plugin instance. BasePlugin takes care of this for you.
+BasePlugin provides a simple wrapper around the Redis API that Plugins should use for storage - it'll handle namespacing the key in the format ``<bot_id>:<channel_id>:<plugin_slug>:<key_name_stripped>``. This ensures that there's no collisions.
```

Comment:
"there's no" *there are :P

Also can you just add "This also means that plugins can't access data from other plugins or other channels."? Thanks!